portfolio/src/main/webapp/script.js

```diff
@@ -25,4 +25,18 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+};
+
+// Tutorial Step 2.
```

Comment:
Could you add a comment for this function explaining what this does in a bit more detail